packages/module/src/FileDropZone/FileDropZone.scss

```diff
@@ -0,0 +1,47 @@
+.pf-chatbot__dropzone {
+  height: 100vh;
+
+  .pf-v6-c-multiple-file-upload__main {
+    background: rgba(224, 224, 224, 0.5);
+    --pf-v6-c-multiple-file-upload__main--BorderWidth: var(--pf-t--global--border--width--action--hover);
+    --pf-v6-c-multiple-file-upload__main--BorderStyle: dashed;
+    --pf-v6-c-multiple-file-upload__main--BorderColor: rgba(0, 102, 204, 1);
+    --pf-v6-c-multiple-file-upload__main--BorderRadius: var(--pf-t--global--border--radius--medium);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .pf-v6-c-multiple-file-upload__title-text {
+    --pf-v6-c-multiple-file-upload__title-text--FontSize: var(--pf-t--global--font--size--heading--h3);
+    --pf-v6-c-multiple-file-upload__title-text--FontWeight: 700;
```

Comment:
There's a variable for this, but it didn't seem to be getting picked up in the build (pf-t--global--font--weight--heading), so I reverted back to the value.

Reply:
hm... it may be because we are not on the latest versions of PF, i'll bump the PF version in my follow up PR and try again to see if we can use the right token 👍🏻